### PR TITLE
Don't take into account beta channels when we have beta disabled

### PR DESCRIPTION
**Pull request by srbarrios**
## What does this PR change?

As we don't have anymore beta channels, we need to skip them from our scenarios and list of channels.
Let's do it programmatically, instead of removing code lines.

## GUI diff

No difference.

- [x] **DONE**

## Documentation
- No documentation needed

- [x] **DONE**

## Test coverage

- Cucumber tests were refactored

- [x] **DONE**

## Links

Ports:
- Manager-4.3

- [ ] **DONE**

## Changelogs

Make sure the changelogs entries you are adding are compliant with https://github.com/uyuni-project/uyuni/wiki/Contributing#changelogs and https://github.com/uyuni-project/uyuni/wiki/Contributing#uyuni-projectuyuni-repository

If you don't need a changelog check, please mark this checkbox:

- [x] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

# Before you merge

Check [How to branch and merge properly](https://github.com/uyuni-project/uyuni/wiki/How-to-branch-and-merge-properly)!

### CODE REVIEW

File: testsuite/features/support/env.rb [hunk 1]
```diff
@@ -58,6 +58,7 @@
 $is_using_build_image = ENV.fetch('IS_USING_BUILD_IMAGE', false)
 $is_using_scc_repositories = (ENV.fetch('IS_USING_SCC_REPOSITORIES', 'False') != 'False')
 $catch_timeout_message = (ENV.fetch('CATCH_TIMEOUT_MESSAGE', 'False') == 'True')
+$beta_enabled = ENV.fetch('BETA_ENABLED', false)
```
Comment (nodeg) [inline testsuite/features/support/env.rb:61]: We need to keep in mind using this in sumaform/susemanager-ci once we have beta channels again.

**Review by nodeg — APPROVED** [review]
Overall, I like this approach. Thank you, Oscar.

File: testsuite/features/support/env.rb (hunk 1 above)
Comment (srbarrios) [reply testsuite/features/support/env.rb:61]: Yup. Or at least, change the default to true.

**Review by Bischoff — APPROVED** [review]
Wow, very clean, with a new environment variable

We will probably need to support this new variable in the future

**Review by mcalmer — APPROVED** [review]
Looks good. This is exactly what we need.
Does sumaform already set "BETA_ENABLED" or do we need a change for that as well?

**Review by NamelessOne91 — APPROVED** [review]
LGTM
Thanks Oscar

**Comment by srbarrios** [comment]
> Looks good. This is exactly what we need. Does sumaform already set "BETA_ENABLED" or do we need a change for that as well?

I'm gonna jump into sumaform now. But as the default is false, we are good for now.

**Pull request merged**